### PR TITLE
pylibscrypt 1.6.1

### PR DESCRIPTION
**Pull request by holgern**
https://pypi.python.org/pypi/pylibscrypt
https://github.com/jvarho/pylibscrypt

### CODE REVIEW

File: recipes/pylibscrypt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+test:
+  imports:
+    - pylibscrypt
+  commands:
+    - conda inspect linkages -p $PREFIX pylibscrypt  # [not win]
+    - conda inspect objects -p $PREFIX pylibscrypt   # [osx]
+
+about:
+  home: https://github.com/jvarho/pylibscrypt
+  license: ISC
```
Comment (CJ-Wright) [inline recipes/pylibscrypt/meta.yaml]: Please add a license family

File: recipes/pylibscrypt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - pip
+    - toolchain
+    - openssl 1.0.*                # [not win]
+  run:
+    - openssl 1.0.*                # [not win]
+    - python
+
+test:
+  imports:
+    - pylibscrypt
```
Comment (CJ-Wright) [inline recipes/pylibscrypt/meta.yaml]: Could you please add some more import tests?

**Comment by holgern** [comment]
OSError: [Errno 28] No space left on device ?
Should i retrigger the build?

**Comment by mwcraig** [comment]
LGTM, merging. Thanks, @holgern

**Pull request merged**